engine/apps/alerts/escalation_snapshot/escalation_snapshot_mixin.py

```diff
@@ -245,6 +245,11 @@ def start_escalation_if_needed(self, countdown=ScenarioStep.CROSS_ACTION_DELAY,
         """
         AlertGroup = apps.get_model("alerts", "AlertGroup")
 
+        is_on_maintenace_or_debug_mode = (
```

Comment:
This is used ~3 times in this PR, so probably makes sense to move it to `AlertReceiveChannel` as a property

Reply:
The problem is that I don't want to move this check to alert_recevie_channel, since it checks organization too. Also, sometimes it's needed to check only debug/maintenance mode, or check it only for organization. You are right, that it's duplication of code, but I don't want to sacrifice SRP principle in favour of DRY. I have a concept of Maintenace v2, which should simplify all concept and code even further  and will require no such checks at all.